src/test/suite/extension.test.ts

```diff
@@ -8,4 +12,15 @@ suite('Extension Test Suite', () => {
 		assert.ok(vscode.extensions.getExtension(extensionId));
 		done();
 	});
+
+	test('by default, didact setting to open on startup should be false', function (done) {
```

Comment:
somehow my comment on the async / done was gone? I see no need in having it because there is nothing async called inside nor using await.

Reply:
I resolved it -- I got rid of async. can I get rid of done too?

Reply:
yes. done is used when running a test asynchronously. so you need to call done once the test has finished. In your case everything is executed synchronously and the done seems obsolete to me.

Reply:
both in the function param and the call inside the test to done()

Reply:
got it